packages/core/package.json

```diff
@@ -2,18 +2,33 @@
   "name": "@readme/api-core",
   "version": "7.0.0-alpha.3",
   "description": "The magic behind `api` 🧙",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    },
+    "./errors/fetchError": {
```

Comment:
This is the only other export because it might be nice for folks to be able to do `instanceof FetchError` in their implementations.